### PR TITLE
aligned individual client details page and camp details page

### PR DESCRIPTION
**Pull request by john-william-cross**

### CODE REVIEW

**Comment by john-william-cross** [comment]
> It looks like the client edit is no longer a modal control as it just shows up at the bottom instead of taking over the screen.

**Comment by john-william-cross** [comment]
> > It looks like the client edit is no longer a modal control as it just shows up at the bottom instead of taking over the screen.

For some reason if you double click it modal control does come up but not if you single click. I'll see if I can figure it out.

**Comment by john-william-cross** [comment]
I will work on it tomorrow afternoon- I will then be out of town until
Monday evening

**Comment by wilsont3** [comment]
Sounds good! Thank you for all you have done this week John. Have a great
weekend!

Tim

**Comment by john-william-cross** [comment]
Submitted a new pull request. Fixed things for admin pages and volunteer
pages

**Comment by john-william-cross** [comment]
Looking at both the master branch and the branch I just submitted, there is
a problem with submitting a new client after creating the client. I'm not
sure what is going on with it, I doubt I have time to figure it out.

**Comment by wilsont3** [comment]
No problem. I can look into it. Thank you and have a great weekend!

File: src/app/client/client-edit-modal/client-edit-modal.component.html [hunk 1]
```diff
@@ -6,16 +6,28 @@ <h4 class="modal-title">Client Details</h4>
   <div class="modal-body">
       <form [formGroup]="clientForm" style="opacity:0.93">
         <div class="col-md-12">
+            <table align="right">
+                <tr>                      
+                  <th><input type="text" placeholder="preferred name" formControlName="preferred_name"/></th>
+                </tr>
+              </table> 
           <label>Preferred Name: </label>
-          <input type="text" placeholder="preferred name" formControlName="preferred_name"/>
```
Comment (wilsont3) [inline src/app/client/client-edit-modal/client-edit-modal.component.html]: Looking online, it appears that the best practice for aligning to the right is to add a style tag with float:right as the value, so this line would be:
`<input type="text" style="float:right" placeholder="preferred name" formControlName="preferred_name"/>`

**Comment by john-william-cross** [comment]
I will look at this tomorrow and get back to you.

File: src/app/client/client-edit-modal/client-edit-modal.component.html (hunk 1 above)
Comment (wilsont3) [reply src/app/client/client-edit-modal/client-edit-modal.component.html]: Another way to do this that I think is actually better is to add `class="float-right"` to the input tag as this does the same thing as above.  `class="float-left"` also does the same thing except to the left.

**Comment by john-william-cross** [comment]
I got this to work. When would you like the changes finished by? I can get
them done by tonight but if it doesn't have to be done right away I'll
finish it by tomorrow evening.

**Comment by wilsont3** [comment]
By tomorrow evening is just fine.

**Comment by john-william-cross** [comment]
Ok, I made the changes and pushed them. Unless I missed something, all the
places I used tables to align right now use float-right. If I did things
correctly they should be reflected in the form alignment november first
pull request.

I took note of some things I noticed while navigating the website. I've
noticed that some of the buttons seem to work in some situations and not
work in other situations... not sure why that would be.


Client-side: "submit client" button doesn’t work on create client screen


Camp details: "get current location" button --> I can’t get it to go to the
right


Search for clients: can’t get "search button" next to "preferred name" to
align right


End route button doesn’t work all the time.


Back button on admin volunteer info page doesn’t work all the time.

**Comment by wilsont3** [comment]
Thank you John! Could you add the issues as cards in the To-Do column?

Tim

**Comment by wilsont3** [comment]
It looks like the files are still the same.  Did you push the changes to this pull request or are they on another branches pull request?

**Comment by john-william-cross** [comment]
Which branch were you looking at? There should have been an updated pull
request for the November first branch sometime last week.

**Pull request closed without merging**